scripts/getRecentUsers.js

```diff
@@ -0,0 +1,46 @@
+require('dotenv').load();
+
+let request = require('request');
+let fs = require('fs');
+
+
+module.exports.run = function run() {
+    
+    let date = new Date();
+    date.setDate( date.getDate() - 1 ); // Remove one day
+    const requestUrl =
+        process.env.API_URL + '/data/game'+
+            '?filter=endTime=gt="'+date.toISOString()+'"'+
+            '&include=playerStats.player'+
+            '&fields[player]=login'+
+            '&sort=-endTime'
+        ;
+     
+	request(
+        requestUrl,        
+        function (error, response, body) {
+            if (! error) {
+                let apiRecentPlayers = JSON.parse(body);
+                let recentPlayers = [];
+
+                if (apiRecentPlayers.included == undefined) return;
+
+                for(let i = 0; i < apiRecentPlayers.included.length; i++) {
```

Comment:
apiRecentPlayer.foreach() ?

Reply:
`for` is used everywhere so I'm going to keep it. However I can certainly improve it yes